cli/cmd/install.go

```diff
@@ -212,157 +239,118 @@ func newCmdInstall() *cobra.Command {
 	return cmd
 }
 
-func validateAndBuildConfig(options *installOptions) (*installConfig, error) {
-	if err := options.validate(); err != nil {
-		return nil, err
-	}
-
-	if options.highAvailability && options.controllerReplicas == defaultControllerReplicas {
-		options.controllerReplicas = defaultHAControllerReplicas
+func (options *installOptions) validate() error {
+	if options.identityOptions == nil {
+		log.Fatal("missing identity options")
```

Comment:
We don't use `log.Fatal` in this package, since all files in the package are being run from the CLI, and the formatting is jarring when the CLI exits with `log.Fatal`. If you need to exit immediately, use `fmt.Println` followed by `os.Exit(1)` (see dashboard.go for an example of that). Otherwise, just return an error and let cobra format it correctly.

Reply:
This is meant to denote a programming error. Would it be better to use `panic!`?

Reply:
Yes, `panic` would be better. We went through this package and painstakingly scrubbed all of the usages of `log.Fatal` a while back, so I want to be vigilant about it creeping back in.